<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="3" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="14335" started-at="2018-07-18T05:23:32Z" finished-at="2018-07-18T05:23:46Z">
    <groups>
      <group name="login">
        <method signature="TMCSourceUILogin.tmcSourceLogin(java.lang.String, java.lang.String)[pri:0, instance:co.testNG.tmcsourceui.TMCSourceUILogin@3a5ed7a6]" name="tmcSourceLogin" class="co.testNG.tmcsourceui.TMCSourceUILogin"/>
      </group> <!-- login -->
    </groups>
    <test name="Default test" duration-ms="14335" started-at="2018-07-18T05:23:32Z" finished-at="2018-07-18T05:23:46Z">
      <class name="co.testNG.tmcsourceui.TMCSourceUILogin">
        <test-method status="PASS" signature="invokeBrowser()[pri:0, instance:co.testNG.tmcsourceui.TMCSourceUILogin@3a5ed7a6]" name="invokeBrowser" is-config="true" duration-ms="6375" started-at="2018-07-18T10:53:32Z" finished-at="2018-07-18T10:53:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invokeBrowser -->
        <test-method status="FAIL" signature="tmcSourceLogin(java.lang.String, java.lang.String)[pri:0, instance:co.testNG.tmcsourceui.TMCSourceUILogin@3a5ed7a6]" name="tmcSourceLogin" duration-ms="1563" started-at="2018-07-18T10:53:41Z" data-provider="getData" finished-at="2018-07-18T10:53:42Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[tmc]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[tmc]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at org.testng.Assert.assertTrue(Assert.java:51)
	at co.testNG.tmcsourceui.TMCSourceUILogin.tmcSourceLogin(TMCSourceUILogin.java:53)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tmcSourceLogin -->
        <test-method status="FAIL" signature="tmcSourceLogin(java.lang.String, java.lang.String)[pri:0, instance:co.testNG.tmcsourceui.TMCSourceUILogin@3a5ed7a6]" name="tmcSourceLogin" duration-ms="1351" started-at="2018-07-18T10:53:42Z" data-provider="getData" finished-at="2018-07-18T10:53:44Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[tm]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[tmc_upload]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at org.testng.Assert.assertTrue(Assert.java:51)
	at co.testNG.tmcsourceui.TMCSourceUILogin.tmcSourceLogin(TMCSourceUILogin.java:53)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tmcSourceLogin -->
        <test-method status="PASS" signature="tmcSourceLogin(java.lang.String, java.lang.String)[pri:0, instance:co.testNG.tmcsourceui.TMCSourceUILogin@3a5ed7a6]" name="tmcSourceLogin" duration-ms="1809" started-at="2018-07-18T10:53:44Z" data-provider="getData" finished-at="2018-07-18T10:53:46Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[tmc]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[tmc_upload]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tmcSourceLogin -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:co.testNG.tmcsourceui.TMCSourceUILogin@3a5ed7a6]" name="closeBrowser" is-config="true" duration-ms="730" started-at="2018-07-18T10:53:46Z" finished-at="2018-07-18T10:53:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- co.testNG.tmcsourceui.TMCSourceUILogin -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
